l10n_fi_payment_terms/README.rst

```diff
@@ -8,8 +8,6 @@ Finnish Invoice Payment Terms
 
 Common Finnish invoice payment terms.
 
-**Please note that this module will remove the default payment terms!**
```

Comment:
Why is `l10n_fi_payment_terms` in the same PR? You seem to have another PR #12 open for almost the same thing.